### PR TITLE
Briankang tpuv5e gemma2b training

### PR DESCRIPTION
**Pull request by brianchunkang**
<br>
Notebook using TPUv5e to fine-tune Gemma. Based on the distributed Gemma tuning example on our Gemma docs: https://ai.google.dev/gemma/docs/distributed_tuning

Requirements to enable this to successfully run during automated testing: Vertex custom model training quota for TPUv5e,
Kaggle tokens and keys supplied for the notebook to execute, and disk size of 200GB+
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb [hunk 1]
```diff
@@ -0,0 +1,1509 @@
+{
```
Comment (gericdong) [inline notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: <p>Update the license year.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (gericdong) [inline notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.        ! gcloud components update --quiet
</pre><p>No need to update gcloud.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (gericdong) [inline notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: <p>populat &gt; populate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (gericdong) [inline notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    SERVICE_ACCOUNT = "your-service-account"  # @param {type:"string"}
</pre><p>Set SERVICE_ACCOUNT = "[your-service-account]" instead. "[your-service-account]" will be replaced by a service account in the integration test.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (brianchunkang) [reply notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: Done

Comment (brianchunkang) [reply notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: gcloud artifacts repositories create didn't recognize the tag --repository-format=docker and it looks like it was due to older components. I can try again without it if you're certain

Comment (brianchunkang) [reply notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: Done

Comment (brianchunkang) [reply notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb:1]: All set

**Comment by gericdong** [comment]
The test is failing on Kaggle access. The integration test currently doesn't have the capability to set the Kaggle key automatically. Opened enhancement b/330199110 to track.

**Review by gericdong — APPROVED** [review]

**Pull request merged**